modules/migration-changing-migration-plan-limits.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-changing-migration-plan-limits_{context}']
+= Changing migration plan limits for large migrations
+
+A single migration plan has the following limits for guidance when performing large scale migrations:
+
+* 10 namespaces. If this limit is exceeded, an error message appears and you cannot create a migration plan.
+* 100 Pods. If this limit is exceeded, a warning message appears.
+* 100 persistent volumes. If this limit is exceeded, a warning message appears.
+
+You can change the default limits by updating the following values in the Migration controller CR:
+
+[source,yaml]
+----
+[...]
+migration_controller: true
+
+# This configuration is loaded into mig-controller, and should be set on the
+# cluster where `migration_controller: true`
+mig_pv_limit: 100
+mig_pod_limit: 100
+mig_namespace_limit: 10
```

Comment:
@jwmatthews 

Can we provide any guidelines for max PVs and Pods? 

Also, if exceeding the namespace limit produces an error message and the user can't create migration plan, why are we allowing the user to change the limit here?

Reply:
Little scale testing has been done up to now.  We have seen success of 1000 namespace migrations.

The reasoning for these limits is to reduce chances of customers hitting issues at scale unexpectedly.  The limits are kind of a communication aide to be cautious.

I don't think we have much more guidance on the numbers to share.

We chose a limit of 100 PVs and 100 Pods as an arbitrary limit we felt would be successful.  Scale testing is focused on confirming these, but we don't expect to have that data for another ~2 weeks.

I would assume as we learn more and enhance scale issues these numbers will be increased in future.